utils/src/main/kotlin/Utils.kt

```diff
@@ -273,7 +273,8 @@ fun normalizeVcsUrl(vcsUrl: String): String {
  */
 fun resolveWindowsExecutable(executable: File): File? {
     val extensions = Os.env["PATHEXT"]?.splitToSequence(File.pathSeparatorChar).orEmpty()
-    return extensions.map { File(executable.path + it) }.find { it.isFile } ?: executable.takeIf { it.isFile }
+    return extensions.map { File(executable.path + it.toLowerCase()) }.find { it.isFile }
```

Comment:
Does the `toLowerCase()` have any effect, for example does this allow to find exectuables that could not be found before? Shouldn't both upper- and lowercase be checked in this case? Or is the case ignored anyways on Windows and this only makes the code longer?

Reply:
No, `toLowerCase()` does not have any such effect, as the Windows file system is case-insensitive. While it makes the code a bit longer, it makes the log output read nicer, like `pip.exe show` instead of `pip.EXE show`. That's because on Windows the members of `PATHEXT` usually are all upper case strings.

Reply:
This makes sense.